static/templates/compose_all_everyone_warning.hbs

```diff
@@ -0,0 +1,9 @@
+{{#> compose_banner }}
+    {{#tr}}
+    Are you sure you want to mention all <strong>{stream_count}</strong> people in this stream?
+    <br />
+    This will send email and mobile push notifications to most of those <strong>{stream_count}</strong> users.
+    <br />
+    If you don't want to do that, please edit your message to remove the <strong>@{wildcard_mention}</strong> mention.
+    {{/tr}}
+{{/compose_banner}}
```

Comment:
I don't know what commit we want to put this in, but we should rename it to something like `compose_banner/wildcard_warning.hbs`? The "all_everyone" label is a very silly way to refer to "wildcard" mentions :).

Reply:
Ah yeah, I think this is the only error message I didn't change the name of to be something more reasonable, because I forgot we call them wildcards! It's a new file, so I think choosing a good name is a good idea.